infrastructure/w3id/tests/w3id.test.ts

```
@@ -0,0 +1,81 @@
+        expect(id1.id).toEqual(id2.id);
+        expect(id1.logs).toBeUndefined();
+    });
+
+    test("ID Generation: Creates IDLogManager", async () => {
+        const id = await new W3IDBuilder()
+            .withRepository(InMemoryStorage.build())
+            .withSigner(createSigner(keyPair))
+            .withNextKeyHash(falso.randText())
+            .build();
+        expect(id.logs).toBeInstanceOf(IDLogManager);
+        const genesisLog = (await id.logs?.repository.findMany({}))[0];
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)